docs/en/setup/backend/backend-telemetry.md

```diff
@@ -19,7 +19,7 @@ telemetry:
 but you can set one of `prometheus` to enable them, for more information, refer to the details below.
 
 ## Self Observability
-
+### Dedicated IP or hostname
```

Comment:
```suggestion
### Static IP or hostname
```

I think Dedicated IP would be assigned by k8s, but the root cause is they are not static, IP would change in every pod reboot or redeployment.